content/en/observability_pipelines/sources/azure_event_hubs.md

```diff
@@ -0,0 +1,176 @@
+
+### Configure Kafka-compatible connection for the Event Hub
+
+Azure Event Hubs exposes a Kafka endpoint at `NAMESPACE.servicebus.windows.net:9093`, which Observability Pipelines uses as the Kafka source.
+
+#### Get the Kafka endpoint
+
+1. In the Azure Portal, navigate to your Event Hubs Namespace (for example, `myeventhubns`).
+1. On the **Overview** page, under the **Essentials** section, locate the **Host name** or **Fully Qualified Domain Name (FQDN)**. It is in the format: `<NAMESPACE>.servicebus.windows.net` (for example, `myeventhubns.servicebus.windows.net`).
+1. Append the Kafka port `:9093` to form the Bootstrap Servers value: `<NAMESPACE>.servicebus.windows.net:9093`.
+    - For example, if your namespace is `myeventhubns`, the Bootstrap Servers is `myeventhubns.servicebus.windows.net:9093`.
+    - You need this information when you set up the Observability Pipelines Kafka source.
+
+#### Set up authentication
+
+1. Azure Event Hubs uses SASL_SSL with the PLAIN mechanism for Kafka authentication.
+1. The connection string is formatted for Observability Pipelines:
+    ```
+    Username: $$ConnectionString
+    Password: Endpoint=sb://<NAMESPACE>.servicebus.windows.net/;SharedAccessKeyName=<PolicyName>;SharedAccessKey=<Key>
+    ```
+
+## Set up a pipeline with the Kafka source
+
+Select your platform.
+
+{{< tabs >}}
+{{% tab "Kubernetes" %}}
+1. Navigate to [Observability Pipelines](https://app.datadoghq.com/observability-pipelines).
+1. Select the Kafka source.
+    1.  In the **Group ID** field, specify or create a unique consumer group (for example, `datadog-consumer-group`).
+    1.  Enter `datadog-topic` in the **Topics** field.
+    1.  Toggle the switch to enable SASL authentication.
+    1.  In the **Mechanism** dropdown menu, select **PLAIN**.
+    1.  Enable TLS.
+        1. Configure your `values.yaml` file to use the certificate that works as part of the container image:
+            ```
+            initContainers:
+            - name: copy-config
+            image: gcr.io/datadoghq/observability-pipelines-worker:latest
+            imagePullPolicy: IfNotPresent
+            command: ['/bin/sh', '-c', 'mkdir -p /config-volume/observability-pipelines-worker/config/ && cp /etc/ssl/certs/ca-certificates.crt /config-volume/observability-pipelines-worker/config/ca-certificates.crt']
+            volumeMounts:
+            - name: config-volume
+                mountPath: /config-volume
+            extraVolumes:
+            - name: config-volume
+            emptyDir: {}
+            extraVolumeMounts:
+            - name: config-volume
+            mountPath: /config-volume
+            ```
+            **Note**: When install the Worker with the install command you need to add:
+            ```
+            --set env[0].name=DD_OP_DATA_DIR,env[0].value='/config-volume/observability-pipelines-worker/'
+            ```
+        1. In the **Certificate path** field, enter `/cert.pem`.
```

Comment:
@ckelner I'm not sure what the user should enter in the UI here.

Reply:
So when they add the init container they chose what to call the file, currently line 101 here: https://github.com/DataDog/documentation/pull/30211/files#diff-bbd5e4a879952a151f7689b8580757bfe890c7ba1f004db9eaeb96eeba6cd2cbR101

> command: ['/bin/sh', '-c', 'mkdir -p /config-volume/observability-pipelines-worker/config/ && cp /etc/ssl/certs/ca-certificates.crt /config-volume/observability-pipelines-worker/config/ca-certificates.crt']

In this case we've not renamed it, so they would enter `/ca-certificates.crt`

Reply:
Link to TLS cert doc